internal/hosts/hosts.go

```diff
@@ -97,6 +97,22 @@ func (n *Host) GetSAPSystems() (map[string]*sapsystem.SAPSystem, error) {
 	return systems, nil
 }
 
+func (n *Host) GetSAPSystemsList() (sapsystem.SAPSystemsList, error) {
+	systems, err := sapsystem.Load(n.client, n.Name())
+	if err != nil {
+		return nil, err
+	}
+
+	systemsList := make(sapsystem.SAPSystemsList, len(systems))
```

Comment:
Why do we have this loop? Cannot we use directly the `systems` map? At the end they have the same content

Reply:
I assume that by "using directly the `systems` map you mean using the `GetSAPSystems` method?
The thing is that if I use `GetSAPSystems` in the template, I can't easily add a comma after each SID but the last one (since in a string map, each key is not an int but a string, so I can't easily figure out the `len-1` case :/... It might be that I'm not familiar enough with templates. A bit more details if you can think of a way of using the string map in the template would help

Reply:
As we just need to split the string by the `,`, why don't we just add the `Split` method to our templating system, so we can use it?
Here an example: https://stackoverflow.com/questions/19771787/golang-how-to-split-string-in-template
We would need to add the func here: https://github.com/trento-project/trento/blob/main/web/layout.go#L142

I see that you need this already 2 times, so it would make sense